index.html

```diff
@@ -1361,18 +1361,16 @@
           [data]  DDDDDDDDDDDDDDDDDDDDDDDDDDDDD...                        D
           [data]  ......
         </pre>
-        <section id='channelpixellayout-interface'>
+        <section id='channelpixellayout-dictionary'>
           <h2>
-            <code>ChannelPixelLayout</code> interface
+            <code>ChannelPixelLayout</code> dictionary
           </h2>
           <p>
             Each <a>channel</a> is represented by an <a>ChannelPixelLayout</a>
```

Comment:
`<dfn>ChannelPixelLayout</dfn>`

(For every interface and its members, there should be corresponding a `<dfn>` in the prose. ReSpec autolinks linkless <a> element to the `<dfn>`s of the same name. ReSpec reports its warning and errors in console as well as in its UI that is floating in the top right corner of the document.)